app/modules/turtleGrapher/datastruct_savvy.py

```diff
@@ -95,16 +109,23 @@ def parse_gene_dict(graph, gene_dict, uriGenome, geneType):
             graph.add((bnode_start, gu('rdf:type'), gu('faldo:ExactPosition')))
             graph.add((bnode_end, gu('rdf:type'), gu('faldo:Position')))
             graph.add((bnode_end, gu('rdf:type'), gu('faldo:ExactPosition')))
-            if gene_record['ORIENTATION'] is '+':
-                graph.add((bnode_start, gu('rdf:type'), gu(
-                    'faldo:ForwardStrandPosition')))
-                graph.add((bnode_end, gu('rdf:type'), gu(
-                    'faldo:ForwardStrandPosition')))
-            else:
-                graph.add((bnode_start, gu('rdf:type'), gu(
-                    'faldo:ReverseStrandPosition')))
-                graph.add((bnode_end, gu('rdf:type'), gu(
-                    'faldo:ReverseStrandPosition')))
+
+            if 'ORIENTATION' in gene_record:
+                if gene_record['ORIENTATION'] is '+':
+                    graph.add((bnode_start, gu('rdf:type'), gu(
+                        'faldo:ForwardStrandPosition')))
+                    graph.add((bnode_end, gu('rdf:type'), gu(
+                        'faldo:ForwardStrandPosition')))
+                else:
+                    graph.add((bnode_start, gu('rdf:type'), gu(
+                        'faldo:ReverseStrandPosition')))
+                    graph.add((bnode_end, gu('rdf:type'), gu(
+                        'faldo:ReverseStrandPosition')))
+
+            if geneType == 'PanGenomeRegion':
```

Comment:
Not sure if this lets us know what a panseq region is in the db.

Reply:
Looks like we assign panseq regions earlier under the var `geneType`

Reply:
Tag should be `:PanGenomeRegion`

Reply:
```
PREFIX : <https://www.github.com/superphy#>
PREFIX g: <http://www.biointerchange.org/gfvo#>
PREFIX so: <http://purl.obolibrary.org/obo/SO_>
PREFIX f: <http://biohackathon.org/resource/faldo#>
SELECT ?s WHERE { ?s a :PanGenomeRegion }
```
Returns blank on my local machine, gonna try testing on a computer that I can let run for awhile.